tests/data/token_indexers/token_feature_indexer.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=no-self-use,invalid-name
```

Comment:
The name of this file needs to include the word "test" for pytest to actually discover that it contains tests.